### PR TITLE
Initial version of configuration files

### PR DESCRIPTION
**Pull request by samshah**
Fully backwards compatible with existing code.

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  popnodes: pop2010
+
+interventions:
+  base_path: COVIDScenarioPipeline/data
+
+  NoNPI: NPI_Scenario1_None.R
+  SC: NPI_Scenario2_School_Closure.R
+  BI1918: NPI_Scenario3_Bootsma_1918Influenza.R
+  KansasCity: NPI_Scenario4_KansasCity.R
+  Wuhan: NPI_Scenario5_Wuhan.R
```
Comment (samshah) [inline config.yml]: plan is to have these be dynamically defined (by globbing a filepath) -- will do in future (not backwards compatible)

Comment (jcblemai) [reply config.yml]: oh great idea

**Comment by jcblemai** [comment]
Very good thank you. Let's just keep in mind (for later) that there will be other way of defining NPIs than R script (each module could be R, python or a file). If it's too much of a hassle rn we can forget that.

I didn't knew about click I'll keep it in mind for my other projects.

**Comment by jwills** [comment]
This looks great Sam-- thank you so much!

**Pull request merged**